client/reader/post-options/_style.scss

```diff
@@ -1,91 +0,0 @@
-.post-options__trigger {
-	display: block;
-	cursor: pointer;
-	color: lighten( $gray, 10 );
-	border-radius: 4px;
-
-	.gridicon__ellipsis {
-		display: block;
-		padding: 2px 4px;
-		fill: lighten( $gray, 10 );
-		transition: all 0.15s cubic-bezier(0.175, 0.885, 0.32, 1.275);
-	}
-
-	.post-options__label {
-		display: none;
-	}
-
-	&:hover {
-		.gridicon__ellipsis {
-			fill: $blue-light;
-		}
-	}
-
-	&.is-triggered {
-		color: $blue-medium;
-
-		.gridicon__ellipsis {
-			fill: $blue-dark;
-			transform: rotate( 90deg );
-		}
-	}
-}
-
-.post-options__hr {
-	margin: 8px 0;
-	background: lighten( $gray, 30 );
-}
-
-.post-options__reblog,
-.post-options__save,
-.post-options__edit,
-.post-options__view {
-	.gridicon {
-		position: absolute;
-		top: 7px;
-		left: 13px;
-		fill: lighten( $gray, 10 );
-		transition: all 0.15s cubic-bezier(0.175, 0.885, 0.32, 1.275);
-	}
-
-	&:hover .gridicon,
-	&:focus .gridicon {
-		fill: $white;
-	}
-}
-
-.popover__menu .follow-button {
```

Comment:
Hrm. This was unscoped, so removing it might have more of an effect than we intend..